examples/README.md

```diff
@@ -0,0 +1,61 @@
+# Examples for Testing Features Locally
+
+The example projects are meant to be used to test features locally by contributors working on this SDK. In the `requirements.txt` file, the deepgram package used is the local version:
+
+```
+-e ../../
+```
+
+## Steps to Test Your Code
+
+If you are contributing changes to this SDK, you can test those changes by using the `prerecorded` or `streaming` projects in the `examples` folder. Here are the steps to follow:
+
+### Add Your Code
+Make your changes to the SDK (be sure you are on a branch you have created to do this work).
+
+### Install dependencies
+
+Run the following command to install the project dependencies:
+
+`pip install -r requirements.txt`
+
+### Edit the API key
+
+Replace the API key where it says 'YOUR_DEEPGRAM_API_KEY'
+
+`DEEPGRAM_API_KEY = 'YOUR_DEEPGRAM_API_KEY'`
+
+### Run the project
+
+Make sure you're in the directory with the `main.py` file and run the project with the following command.
+
+`python3 main.py`
+
+## How to verify that you're testing the local changes
```

Comment:
It may be useful to explain to the user how to install the Deepgram SDK from a local repo. We can do that using:

```bash
pip uninstall deepgram-sdk  # If it's already installed
cd /path/to/deepgram-python-sdk/
pip install -e .
```

This will install the SDK from the user's local source code, which allows them to make changes that are instantly usable in their local examples.

Reply:
So this means the person would uninstall the sdk, then go into the local deepgram sdk (I guess at the root of the project) and do the installation again with the -e flag. So the -e flag is what makes it install the local version?

Reply:
Yeah exactly - it's a really nice way to use packages that you've created (or are editing) locally. The package needs to have a `setup.py` file, which is also required for pypi and the Deepgram SDK has it already.